config/randomMap.json

```diff
@@ -0,0 +1,59 @@
+{
+  "terrain" :
+  {
+    "undergroundAllow" : ["lava"], //others to be replaced by subterranena
+    "groundProhibit" : ["subterranean"] //to be replaced by dirt
+  },
+  "waterZone" :
+  {
+    "treasure" :
+    [
+        { "min" : 2000, "max" : 6000, "density" : 5 },
```

Comment:
You got the values the other way round: 
> Value 100-1000: Frequency of occurence 5.
> Value 2000-6000: Frequency of occurence 1.

Reply:
Does order make any difference?

Reply:
hm, is there transformation or something? I don't see frequency mentions in code and maps look ok as well.
@DjWarmonger , could you please elaborate more?

Reply:
Value 2000-6000: Frequency of occurence 1.
vs
 { "min" : 2000, "max" : 6000, "density" : 5 }

Reply:
I've got it! A typo... @DjWarmonger , thanks for catching that, really nice one!